introduction.md

```diff
@@ -0,0 +1 @@
+Hi. My name is Peter David Carter and I am primarily a web developer with a particular interest in web robots... SkyNets at dawn, gentlemen...

```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.